compile errors from zproto_client_c

I am getting compile errors from the hello_client.xml example:

gsl -trace:1 hello_client.xml
GSL/4.1c Copyright (c) 1996-2016 iMatix Corporation
gsl/4 I: Processing hello_client.xml...
gsl/4 M: Generating stub for connect_to_server...
cc    -c -o hello_client.o hello_client.c
In file included from hello_client.c:39:0:
hello_client_engine.inc:626:1: error: conflicting types for ‘hello_client_set_verbose’
 hello_client_set_verbose (hello_client_t *self, bool verbose)
 ^
In file included from hello_client.c:18:0:
./hello_client.h:62:5: note: previous declaration of ‘hello_client_set_verbose’ was here
     hello_client_set_verbose (bool verbose);
     ^
hello_client.c: In function ‘hello_client_test’:
hello_client.c:71:5: error: ‘self’ undeclared (first use in this function)
     self->verbose = verbose;
     ^

Looking into zproto_client_c I see the signatures for *_set_verbose do mismatch:
$ grep _set_verbose /usr/local/bin/zproto_client_c.gsl
    $(class.name)_set_verbose (bool verbose);
$(class.name)_set_verbose ($(class.name)_t *self, bool verbose)

and "self" is undeclared within $(class.name)_test (bool verbose)

I have the most recent code from github. Did I use zproto_client_c incorrectly?

thanks,
Jeff

--- a/src/zproto_client_c.gsl
+++ b/src/zproto_client_c.gsl
@@ -530,7 +530,7 @@ $(header.:)
 .endfor
 //  Enable verbose tracing (animation) of state machine activity.
 $(CLASS.EXPORT_MACRO)void
-    $(class.name)_set_verbose (bool verbose);
+    $(class.name)_set_verbose ($(class.name)_t *self, bool verbose)
 
 //  Self test of this class
 $(CLASS.EXPORT_MACRO)void

